ProgressBar.js

```diff
@@ -18,53 +19,49 @@ var styles = StyleSheet.create({
   }
 });
 
-
 var ProgressBar = React.createClass({
 
-  mixins: [tweenState.Mixin],
-
   getDefaultProps() {
     return {
       style: styles,
-      easing: tweenState.easingTypes.easeInOutQuad,
+      easing: Easing.inOut(Easing.ease),
       easingDuration: 500
     };
   },
 
   getInitialState() {
     return {
-      progress: 0
+      progress: new Animated.Value(0)
     };
   },
 
   componentDidUpdate(prevProps, prevState) {
     if (this.props.progress >= 0 && this.props.progress != prevProps.progress) {
```

Comment:
This comparison doesn't make sense since `this.props.progress` is `Animated.Value` not float. Any ideas?